resources/references/core-reference/administration-reference/utils.md

```diff
@@ -0,0 +1,109 @@
+# Utils reference
```

Comment:
You have a lot of broken markdown syntax in this article, e.g missing closing of table lines. Please don't forget to scan your article before pushing.

Reply:
I thought i wouldn't need to close of table lines, well